x/examples/smart-proxy/config_broken.yaml

```
@@ -23,5 +23,19 @@ tls:
   - tlsfrag:1
 
 fallback:
-  # Nonexistent server
+  # Nonexistent Outline Server
   - ss://Y2hhY2hhMjAtaWV0Zi1wb2x5MTMwNTprSzdEdHQ0MkJLOE9hRjBKYjdpWGFK@1.2.3.4:9999/?outline=1
+  # Nonexistant Psiphon Config JSON. Not yet supported
+  - {
```

The format I had in mind was like this:
```suggestion
  - psiphon: {
```

This way we can have add other types.

---

done